cowprotocol/cowswap
# 1609/slippage gone from confirmation modal

# Summary

Fixes #1609 

Slippage should be displayed in the confirmation modal

  # To Test

1. Select a pair, click swap
* Slippage is shown
2. Turn on expert mode
* Slippage should be displayed in the swap screen

Play around with the combinations of expert mode, slippage auto, custom slippage, turn on eth-flow, etc
